torch_np/tests/numpy_tests/core/test_scalarmath.py

```diff
@@ -302,7 +306,7 @@ def test_float_modulus_exact(self):
     def test_float_modulus_roundoff(self):
         # gh-6127
         dt = np.typecodes['Float']
-        for op in [floordiv_and_mod, divmod]:
+        for op in [floordiv_and_mod]:  # TODO divmod is not implemented
```

Comment:
assert it below so that it starts failing when we implement it or simply xfail it as the previous test.